src/linear_algebra/general_linear_group.lean

```diff
@@ -199,8 +199,8 @@ lemma coe_to_GL_pos_det (g : special_linear_group n R) :
 
 variable [fact (even (fintype.card n))]
 
-@[simp] lemma coe_GL_pos_neg (g : special_linear_group n R) : ↑(-g) = -(↑g : GL_pos n R) :=
-by {ext, refl}
+@[simp, norm_cast] lemma coe_GL_pos_neg (g : special_linear_group n R) : ↑(-g) = -(↑g : GL_pos n R)
+:= by {ext, refl}
```

Comment:
Mathlib style never puts `:=` at the start of a line, so you should probably line wrap this after the `:`

Reply:
Oops yes sorry!